sdk_generator/outputs/github_parser.json

```diff
@@ -12653,6 +12653,15 @@
   },
   "show environment": {
     "ios": {
+      "asr901": {
```

Comment:
there is ASR920 as well. so I think better to have this folder for abstraction as `asr900`

Reply:
just in case, this is suggesting to change folder name from `asr901` to `asr900`.

src/genie/libs/parser/ios/asr901/show_environment.py
to
src/genie/libs/parser/ios/asr900/show_environment.py

Reply:
ASR920 is an IOS-XE based platform.
ASR901 is the only ASR900 platform based on IOS